doc/nrf/libraries/security/bootloader/bl_storage.rst

```diff
@@ -22,7 +22,7 @@ The library has the following functions for either reading or writing, or in som
 * Invalidation tokens, used to revoke public keys.
 * Additional public key metadata.
 
-The library uses either the OTP region of the User information configuration registers (UICR), when present on SoCs like the nRF9160 or nRF5340, or the internal flash memory.
+The library uses either the OTP region of the User information configuration registers (UICR), when present on SoCs like the nRF91 Series or nRF5340, or the internal flash memory.
```

Comment:
```suggestion
The library uses either the OTP region of the user information configuration registers (UICR), when present on nRF91 Series or nRF5340 devices, or the internal flash memory.
```

Reply:
Is it ok to say it like this?
Was just trying to avoid mentioning SoCs/SiPs, because nRF91 Series are SiPs and not SoCs. nRF5340 is SoC.

Reply:
I expect that this would be OK, but would like a comment from @oyvindronningstad  or @hakonfam.

Reply:
LGTM

Reply:
Thanks @oyvindronningstad, can you check this PR as well? When you have a moment.